src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OxoGReadCounts.java

```diff
@@ -65,55 +59,37 @@ public void annotate(final ReferenceContext refContext,
         Utils.nonNull(gb, "gb is null");
         Utils.nonNull(vc, "vc is null");
 
-        if (g == null || !g.isCalled() || likelihoods == null || !vc.isSNP()) {
+        if (g == null || likelihoods == null) {
             return;
         }
 
-        final Allele ref = vc.getReference();
-        final Allele alt = vc.getAlternateAllele(0);
-
-        int alt_F1R2 = 0;
-        int alt_F2R1 = 0;
-        int ref_F1R2 = 0;
-        int ref_F2R1 = 0;
-
-        for (final ReadLikelihoods<Allele>.BestAllele bestAllele : likelihoods.bestAlleles(g.getSampleName())) {
-            final GATKRead read = bestAllele.read;
-            if (bestAllele.isInformative() && isUsableRead(read) && read.isPaired()) {
-                final Allele allele = bestAllele.allele;
-                if (allele.equals(ref, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        ref_F2R1++;
-                    } else {
-                        ref_F1R2++;
-                    }
-                } else if (allele.equals(alt, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        alt_F2R1++;
-                    } else {
-                        alt_F1R2++;
-                    }
-                }
-            }
-        }
+        final Map<Allele, MutableInt> f1r2Counts = likelihoods.alleles().stream()
+                .filter(Allele::isNonReference)
+                .collect(Collectors.toMap(a -> a, a -> new MutableInt(0)));
 
-        final double numerator;
-        if (ref.equals(REF_C) || ref.equals(REF_A)) {
-            numerator = alt_F2R1;
-        } else {
-            numerator = alt_F1R2;
-        }
-        final double denominator =  alt_F1R2 + alt_F2R1;
-        final double fraction = numerator/denominator;
-
-        gb.attribute(GATKVCFConstants.OXOG_ALT_F1R2_KEY, alt_F1R2);
-        gb.attribute(GATKVCFConstants.OXOG_ALT_F2R1_KEY, alt_F2R1);
-        gb.attribute(GATKVCFConstants.OXOG_REF_F1R2_KEY, ref_F1R2);
-        gb.attribute(GATKVCFConstants.OXOG_REF_F2R1_KEY, ref_F2R1);
-        gb.attribute(GATKVCFConstants.OXOG_FRACTION_KEY, fraction);
+        final Map<Allele, MutableInt> f2r1Counts = likelihoods.alleles().stream()
+                .filter(Allele::isNonReference)
+                .collect(Collectors.toMap(a -> a, a -> new MutableInt(0)));
+
+        Utils.stream(likelihoods.bestAlleles(g.getSampleName()))
```

Comment:
@takutosato I haven't put the ref allele back in because we're still waiting to hear back from CGA.